.github/workflows/sbom.yaml

```diff
@@ -0,0 +1,78 @@
+name: Generate SBOM
```

Comment:
🔧 It would be useful to add some documentation at the top of the workflow, explaining what the goal of this workflow is and some general notes on how it works.